SYCL/ESIMD/ext_math.cpp

```diff
@@ -0,0 +1,220 @@
+//==---------------- ext_math.cpp  - DPC++ ESIMD extended math test --------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// REQUIRES: gpu
+// UNSUPPORTED: cuda
+// RUN: %clangxx-esimd -fsycl %s -o %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+
+// This test checks extended math operations.
+
+#include <CL/sycl.hpp>
+#include <CL/sycl/INTEL/esimd.hpp>
+#include <iostream>
+
+using namespace cl::sycl;
+using namespace sycl::INTEL::gpu;
+
+class ESIMDSelector : public device_selector {
```

Comment:
`ESIMDSelector` and `createExceptionHandler` can be included from the esimd test helper header.